src/tailscale/tailscale.py

```diff
@@ -20,6 +20,8 @@
 )
 from .models import Device, Devices
 
+VERSION = metadata.version(__package__)
```

Comment:
Let's remove custom agent support completely (as I have done in my other packages as well).